src/webgpu/api/validation/render_pipeline/inter_stage.spec.ts

```diff
@@ -1,10 +1,337 @@
 export const description = `
-TODO: interface matching between vertex and fragment shader validation for createRenderPipeline:
-    - superset, subset, etc.
+Interface matching between vertex and fragment shader validation for createRenderPipeline:
+
+[1] TODO(dawn:1448) replace with t.device.limits.maxInterStageShaderVariables
```

Comment:
so non-chromium contributors know how to find this bug:
```suggestion
[1] TODO(crbug.com/dawn/1448) replace with t.device.limits.maxInterStageShaderVariables
```